libs/ui/components/pickers/project-color-picker/color-button/index.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { useState  } from 'react';
+import { Box } from '@chakra-ui/react';
+import Icon from '../../../contents/icon';
+import ColorButtonProps from './props';
+
+export const ColorButton = ({ color, isSelected, setCurrentSelectedColor } : ColorButtonProps) => {
+    //Attributes
+    const [isHovered, setIsHovered] = useState(false);
```

Comment:
```suggestion
    const [isHovered, setIsHovered] = useState<boolean>(false);
```
Always type your states